### PR TITLE
create industry strength benchmarking and evaluation tool section

### PR DESCRIPTION
**Pull request by zhimin-z**
This perfectly solve https://github.com/EthicalML/awesome-production-machine-learning/issues/386

### CODE REVIEW

**Comment by al-yakubovich** [comment]
This section would be great to have!

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This repository contains a curated list of awesome open source libraries that wi
 | [⚔ Adversarial Robustness](#adversarial-robustness) | [💾 Data Storage Optimization](#data-storage-optimisation) | [📓 Data Science Notebook](#data-science-notebook) |
 | [🔥 Neural Search](#neural-search) | [👁️ Industry-strength Computer Vision](#industry-strength-cv) | [🔠 Industry-strength Natural Language Processing](#industry-strength-nlp) |
 | [🍕 Industry-strength Reinforcement Learning](#industry-strength-rl) | [📊 Industry-strength Visualisation](#industry-strength-visualisation) | [🙌 Industry-strength Recommender System](#industry-strength-recsys) |
-| [💰 Commercial Platform](#commercial-platform) |
+| [📈 Industry-strength Evaluation](#industry-strength-evaluation) | [💰 Commercial Platform](#commercial-platform) |
```
Comment (axsaucedo) [inline README.md:25]: Let's change to "Benchmarking & Evaluation", other than that looks good

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Addition looks good as discussed, only change on section naming

File: README.md (hunk 1 above)
Comment (zhimin-z) [reply README.md]: Done! @axsaucedo

Comment (axsaucedo) [reply README.md]: Great - appreciated

**Review by axsaucedo — APPROVED** [review]
Great addition

**Pull request merged**